index.js

```diff
@@ -1,64 +1,62 @@
 'use strict';
 
-var assign = require('object.assign');
+const assign = require('object.assign');
+const { declare } = require('@babel/helper-plugin-utils');
 
-var modules = [require('babel-plugin-transform-es2015-modules-commonjs'), {
-  strictMode: true, // add "use strict"
-  strict: false // this allows __esModule to be exported
-}];
-
-var defaultTargets = {
+const defaultTargets = {
   android: 30,
   chrome: 35,
   edge: 14,
   explorer: 9,
   firefox: 52,
   safari: 8,
-  ucandroid: 1
+  ucandroid: 1,
 };
 
 function buildTargets(options) {
   return assign({}, defaultTargets, options.additionalTargets);
 }
 
-module.exports = function buildAirbnbPreset(context, options) {
-  var transpileTargets = (options && options.targets) || buildTargets(options || {});
+module.exports = declare((api, options) => {
+  api.assertVersion(7);
+
+  const transpileTargets = options.targets || buildTargets(options);
+
+  const debug = typeof options.debug === 'boolean' ? !!options.debug : false;
```

Comment:
`!!options.debug` seems unnecessary after checking `typeof options.debug === 'boolean'`.

Should debug just be `!!options.debug`?

Reply:
the current logic means that if it's not a boolean, it's `false` - i agree that `‼options.debug` isn't needed and it could just be `options.debug`.